### DISCUSSION TITLE
feature: Return old context menu on Windows 11

### DISCUSSION THREAD

**Discussion by krax1337** [farag2/Sophia-Script-for-Windows#267] [Q&A] [answered]
Привет! Можешь рассмотреть возможность включение опции возвращение старого контекстного меню на Windows 11. Это делается через редактирование параметров реестра.

Примеры:

> https://www.pcgamer.com/windows-11-context-menu-fix-right-click/
> https://appuals.com/disable-show-more-options-windows-11/

**Comment by farag2** [comment, marked as answer]
Already in development for the 6.0.6 release.
